From 5a5cc04e1d2cd477fe8e7aa77df50ec890ffdc66 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Wed, 29 Apr 2015 14:18:34 +0800
Subject: [PATCH] MLK-10773-2 ARM: dts: imx6ul: remove power-domains in pxp
 node

Just as the lcdif node, the power-domains property should be
removed to avoid hang issue for pxp driver.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
---
 arch/arm/boot/dts/imx6ul.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6ul.dtsi b/arch/arm/boot/dts/imx6ul.dtsi
index 6b240f6..8937f9b 100644
--- a/arch/arm/boot/dts/imx6ul.dtsi
+++ b/arch/arm/boot/dts/imx6ul.dtsi
@@ -842,7 +842,6 @@
 				clocks = <&clks IMX6UL_CLK_PXP>,
 					 <&clks IMX6UL_CLK_DUMMY>;
 				clock-names = "pxp-axi", "disp-axi";
-				power-domains = <&gpc 2>;
 				status = "disabled";
 			};
 
-- 
1.8.0

